reana/cli.py

```diff
@@ -1058,6 +1062,8 @@ def docker_build(user, tag, component, build_arg, no_cache):  # noqa: D301
             if no_cache:
                 cmd += ' --no-cache'
             cmd += ' -t {0}/{1}:{2} .'.format(user, component, tag)
+            if back_ground:
+                cmd += ' > /dev/null 2>&1 & '
```

Comment:
The `-q` option makes the build to be nicely quiet, displaying only the final SHA1 of the image:

```console
$ docker build -q -t reanahub/reana-job-controller:latest . 
sha256:345bfa584a07ab1e5a8a518b94c23d3f2a7a480a5880f0e885301e5a6110c55b
```